SU2_CFD/include/driver_structure.hpp

```diff
@@ -355,12 +356,12 @@ class CSingleZoneDriver : public CDriver {
 
 
 /*!
- * \class CMultiZoneDriver
+ * \class CFluidDriver
```

Comment:
Probably you are right. Maybe its no problem at all. Don't get my comment wrong, I just don't know what the best way would be. I am fine with this as well!